datafusion/core/benches/sql_planner.rs

```diff
@@ -194,6 +196,16 @@ fn criterion_benchmark(c: &mut Criterion) {
         b.iter(|| physical_plan(&ctx, "SELECT c1 FROM t700"))
     });
 
+    // Test simplest
+    c.bench_function("logical_select_all_from_1000", |b| {
+        b.iter(|| logical_plan(&ctx, "SELECT * FROM t1000"))
```

Comment:
Obviously 1000 > 700, but I'm wondering if we should just reuse the t700 table which already intents to be a "huge" table?

So the query would be "SELECT * FROM t700"